setup.py

```diff
@@ -32,7 +32,7 @@ def get_git_revision_hash():
 INSTALL_REQUIRES = [
     'Markdown>=2.6,<2.7',
     'bleach>=1.4.2,<2.2',
-    'Wagtail<2.3',
+    'Wagtail>=1.10,<2.5',
```

Comment:
hmm... people reported this works with 2.5 too

Reply:
what about `<=2.5`. or `<2.5.1` (current)
I'd like to remove this dependancy altogether.

Reply:
why not `>=1.10`?

if a new Wagtail version breaks this, people will shout and then we can address it
wagtailmedia has `>2.0`

Reply:
Yea probably the best way forward. Thanks @zerolab

Reply:
thinking more of this, let's go `>=2.0`

Reply:
Happy with that, just need to address travis